project-base/app/config/parameters.yml.dist

```diff
@@ -22,3 +22,4 @@ parameters:
     trusted_proxies:
         - "127.0.0.1"
     env(REDIS_PREFIX): ''
+    build-version: "0000000000000000"
```

Comment:
Maybe the default parameter `build-version` in `project-base` is really not needed as the Symfony should use the parameter from `framework` package. During testing I didn't realize that I used old framework in project-base, so the build failed during `composer install` (the parameter was obviously missing).... During tests it seems reasonable to branch-split the monorepo and try it without default parameter in project-base.

Reply:
Okay, I did some digging and we need to have a build-version specified in both files. Composer install fails when build-version is not in parameters file, because it needs to be known before registering the snc_redis bundle. 
And for `RedisVersionFacade` we need a default value in a framework (what if someone don't do this upgrade).